test/coin/collect_coins.cpp

```diff
@@ -1,7 +1,6 @@
 #include <coin.h>
 #include <gs.h>
-
-#include <cassert>
+#include "../assert.h"
```

Comment:
relative includes are kind of bad practice

Reply:
I can't work out how to include the directory with the test.h file using the custom check target

Reply:
you use angle braces and absolute paths. will work fine if the root proj dir (or otherwise) is correctly set in cmake via a `target_include_directories` call or similar

Reply:
We're not including the root project dir, just the src and include dirs

Reply:
Thanks